Flick/Views/ListPreviewView.swift

```diff
@@ -62,6 +65,27 @@ class ListPreviewView: UIView {
         }
     }
 
+    private func setupMediaImages() {
+        for i in 0..<firstThreeMedia.count {
```

Comment:
Maybe just make I more descriptive here? I'm also questioning a switch here haha but I'm not really picky. If it works it works haha.